Ginger/Ginger/Actions/_Common/ActUIElementLib/ActUIElementEditPage.xaml.cs

```diff
@@ -60,16 +60,43 @@ public ActUIElementEditPage(ActUIElement act)
             mPlatform = PlatformInfoBase.GetPlatformImpl(act.Platform);
             List<eLocateBy> LocateByList = mPlatform.GetPlatformUIElementLocatorsList();
             ElementLocateByComboBox.BindControl(mAction, nameof(ActUIElement.ElementLocateBy), LocateByList);
-            ElementTypeComboBox.BindControl(mAction, nameof(ActUIElement.ElementType), mPlatform.GetPlatformUIElementsType());
+
+            //if widgets element, only supported to java platform now.
+            if (act.Platform.Equals(ePlatformType.Java))
+            {
+                ShowWidgetsElementCheckBox();
+            }
+
+            BindElementTypeComboBox();
+
             SetLocateValueFrame();
             ShowPlatformSpecificPage();
             ShowControlSpecificPage();
             ElementLocateByComboBox.SelectionChanged += ElementLocateByComboBox_SelectionChanged;
         }
 
+        private void BindElementTypeComboBox()
+        {
+            ElementTypeComboBox.Items.Clear();
+            if (Convert.ToBoolean(mAction.GetInputParamValue(Fields.IsWidgetsElement)))
```

Comment:
for web isWidgetElement will be null so need to test and confirm Conver.ToBoolean is not throwing exception with null value